build/config/compiler/BUILD.gn

```diff
@@ -283,9 +283,12 @@ config("compiler") {
 
     cflags += common_mac_flags
 
-    # Without this, the constructors and destructors of a C++ object inside
-    # an Objective C struct won't be called, which is very bad.
-    cflags_objcc = [ "-fobjc-call-cxx-cdtors" ]
+    cflags_objcc = [
+      "-std=c++11",
```

Comment:
-fno-exceptions is in //build/config/gcc/BUILD.gn in a config called no_exceptions which is pulled into //build/config/BUILDCONFIG.gn's _native_compiler_settings which goes into the defaults for each target type (executable, static_library, etc). the no_exceptions config only adds it to cflags_cc. maybe if we want that for objc++ (I assume we do?) we should modify //build/config/gcc/BUILD.gn

you can do 'gn desc out/ios_sim_Debug/ //sky/services/ns_net' to see the ccflags_XXX that apply to a particular target and add --blame to see the provenience of those flags